android/app/src/main/res/layout/fragment_login.xml

```diff
@@ -24,38 +24,69 @@
             android:textColor="@color/black"
             android:textSize="36sp" />
 
-        <EditText
-            android:id="@+id/usernameEditText"
+        <TextView
+            android:id="@+id/loginEmailTextView"
```

Comment:
We agreed on last meeting that we will name the id's of UI elements as "type of ui element + purpose of element" in camel case. For this example it should be "textViewEmail". Can you rename these fields?